### DISCUSSION TITLE
Slow down issue

### DISCUSSION THREAD

**Discussion by Trickyeah** [seerge/g-helper#1551] [General]
Hi Seerge or person reading this,

I've noticed a weird issue after my install of g-helper where it doesn't matter what application, what mode, plugged in or on battery, where there will be a random slow down lasting for 2-3 seconds effecting audio and video. I'd describe it as if you play an emulator and the frames drop super low. It happens on all applications including games, FL Studios, Youtube, Spotify, etc . It happens at very random times and doesn't seem to be tied to any specific setting. If you have any suggestions or ideas, as I'm not sure if its related with your program or with the hardware itself, I'd greatly appreciate it.
 (To add; before my laptop would have a problem with freezing, requiring a hard reset, and it seems the crashing has possibly been replaced by this issue as I haven't had many if any freezes, but I'm unsure)

(ASUS G15 GA503RM 2022, Ryzen 9 6900hs, rtx 3060)

**Comment by seerge** [comment]
@Trickyeah hello, I don't think that stutters or slow down anyhow related to the app :) as it's not a driver and not your windows 

But my advice would be clean reinstall of your GPU (and may be Audio) drivers using DDU app https://www.guru3d.com/download/display-driver-uninstaller-download/ as it's often drivers that can cause things like this

**Reply by Trickyeah** [reply]
@seerge Okay thank you, I had done it and we'll see where that goes. I haven't noticed anyone speaking on this but I've noticed an issue, maybe with my model or with windows itself but turbo mode is very broken (as in stuttering in all applications, not related to throttling). I had read through your notes that silent, balanced and turbo are taken directly from the bios itself and I was having the same issues with AC too. I had done a clean install of bios and the problem has continued. As I don't like running my computer hot as only balanced works, have you or someone you've helped run into this issue? I think I may have to replace the system because it seems to be hardware related. I also wanted to say thank you for your hard work and support on this project! You have helped a lot of people!

**Comment by alexatthehub** [comment]
I've got a Flow with the same processor. Your slow-downs may be related to TPM. Asus and AMD have supposedly sorted it out, but even with the latest BIOS and drivers, I was experiencing the periodic stutter you seem to be describing. If you're all up to date, and don't use bitlocker, you could give this a try: find "tpm.sys" in your system32 folder, take ownership of it, enable reading & writing for your user account, and rename it to something like "tpm.sys.old"; this will effectively cripple TPM after a restart but it's been smooth sailing for me since. Be warned, if you do this, anything that relies on TPM will not work, and you could lose access to encrypted files or more.

**Reply by Trickyeah** [reply]
@alexatthehub Thanks for the response,
I had given that an attempt, but it seemed to caused more issues then it was taking away. I appreciate you responding. I have the ability I'm going to replace the laptop entirely and id rather not troubleshoot for the next 3 years, its sad such good potential has been wasted on this laptops endless issues
